version.cmake

```diff
@@ -1,4 +1,4 @@
 # Copyright (C) 2017 Prism Skylabs
 set(ConnectSDK_VERSION_MAJOR 1)
-set(ConnectSDK_VERSION_MINOR 9)
-set(ConnectSDK_VERSION_REVISION 1)
+set(ConnectSDK_VERSION_MINOR 10)
```

Comment:
connect/master already has version 1.10.0.
Have you updated master from remote before branching from it?